src/Services/GTT/shared/GTT.Application/Interfaces/Repositories/IGroupRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using GTT.Application.Response;
+using GTT.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GTT.Application.Interfaces.Repositories
+{
+    public interface IGroupRepository
+    {
+        Task<ListGroupResponse> GetAllGroup(int pageSize, int pageIndex, string filter);
```

Comment:
I think the name GetGroups is more descriptive. as discussed before, we will create a separated API for search. so remove filter param